core/src/display_object/edit_text.rs

```diff
@@ -196,6 +200,11 @@ impl<'gc> EditText<'gc> {
     // This seems to be OS-independent
     const INPUT_NEWLINE: char = '\r';
 
+    /// Maximum duration between two clicks for them to be considered as a multi-click.
+    ///
+    /// Derived experimentally to be 250 ms.
+    const MULTI_CLICK_MAX_DURATION: TimeDelta = TimeDelta::milliseconds(250);
```

Comment:
That value is pretty low, usually 500 ms is used. I'm wondering whether in the future we should add an option to increase that as an accessibility setting.

Reply:
Seems to be 500ms for me in FP on Windows

Reply:
Good shout - yes it is.

Reply:
(I suspect this is actually not handled by edittext, but a general player event "double click" - which edittext listens to. Triple click is probably custom though.

Reply:
That seems to be platform-dependent then, because on GNOME it's constant 250 ms irrespective of double click delay setting.

Unfortunately it seems that winit does not support double clicking though. Also, I wouldn't be able to test the behavior of triple clicking as I cannot change the delay 😐

Reply:
I changed the delay to 500ms and added a TODO. I will address supporting double clicks at OS level in another PR. It should be easy to implement on the web, but on desktop I'm afraid we will need to add a setting until it's implemented in winit.

Reply:
Never mind, I implemented a rough framework for double clicks, so that the click index may be supplied in player events, and when it's not, it's calculated.

Currently it's not supplied and TODOs are added both in desktop and web (tried `js_event.detail` but it doesn't work OOTB).